py/pykms/pykmsutil.cpp

```diff
@@ -26,10 +26,13 @@ void init_pykmstest(py::module &m)
 			.def("reserve_connector", (Connector* (ResourceManager::*)(const string& name))&ResourceManager::reserve_connector,
 			     py::arg("name") = string())
 			.def("reserve_crtc", &ResourceManager::reserve_crtc)
-			.def("reserve_plane", &ResourceManager::reserve_plane,
+			.def("reserve_plane", (Plane* (ResourceManager::*)(Crtc*, Plane::Type, PixelFormat)) &ResourceManager::reserve_plane,
 			     py::arg("crtc"),
```

Comment:
My point was, in the previous commit you break the compilation of this file, because only in this commit you fix the reserve_plane bindings. Well, I didn't test, but I'm quite sure the compiler complains about ambiguous reference.